wdl/transforms/draft3/src/test/scala/wdl/draft3/transforms/expression/UnaryAndBinaryOperatorsEvaluatorSpec.scala

```diff
@@ -58,7 +66,11 @@ class UnaryAndBinaryOperatorsEvaluatorSpec extends FlatSpec with Matchers{
     ("5 - 5", Subtract(fiveLiteral, fiveLiteral), WomInteger(0), WomIntegerType),
     ("5 * 5", Multiply(fiveLiteral, fiveLiteral), WomInteger(25), WomIntegerType),
     ("25 * 5", Divide(PrimitiveLiteralExpressionElement(WomInteger(25)), fiveLiteral), WomInteger(5), WomIntegerType),
-    ("27 % 5", Remainder(PrimitiveLiteralExpressionElement(WomInteger(27)), fiveLiteral), WomInteger(2), WomIntegerType)
+    ("27 % 5", Remainder(PrimitiveLiteralExpressionElement(WomInteger(27)), fiveLiteral), WomInteger(2), WomIntegerType),
+
+    ("10 + 1", Add(tenBigDecimalLiteral, oneBigDecimalLiteral), womBigDecimalEleven, WomBigDecimalType),
+    ("10 * 1", Multiply(tenBigDecimalLiteral, oneBigDecimalLiteral), womBigDecimalTen, WomBigDecimalType),
+    ("10 - 1", Subtract(tenBigDecimalLiteral, oneBigDecimalLiteral), womBigDecimalNine, WomBigDecimalType)
```

Comment:
I feel lack of test cases. 
I can see `mod`,` equals` and several others operations remaining

Reply:
Yes, thought about it. Will add more.

Reply:
I added some tests, think now there is enough cases to be sure that everything works correctly.